packages/sit-onyx/.storybook/preview.ts

```diff
@@ -1,25 +1,34 @@
 import { createPreview } from "@sit-onyx/storybook-utils";
+import type { Preview } from "@storybook/vue3";
 import docsTemplate from "./docs-template.mdx";
 
-import "../src/styles/index.scss";
 import "@sit-onyx/storybook-utils/style.css";
+import "../src/styles/index.scss";
 import "./docs-template.scss";
 
-const preview = {
-  // we need to destructure here because as of Storybook 7.6
-  // it can not statically analyze that the `preview` variable is an object
-  ...createPreview({
-    parameters: {
-      docs: {
-        page: docsTemplate,
-        toc: {
-          title: "Table of Contents",
-          // add our custom "Properties, Events and Slots" headline from docs-template.mdx to the table of contents
-          headingSelector: ".sb-anchor > h3, #properties-events-and-slots, #examples",
-        },
+const basePreview = createPreview({
+  parameters: {
+    docs: {
+      page: docsTemplate,
+      toc: {
+        title: "Table of Contents",
+        // add our custom "Properties, Events and Slots" headline from docs-template.mdx to the table of contents
+        headingSelector: ".sb-anchor > h3, #properties-events-and-slots, #examples",
+      },
+    },
+  },
+});
+
+const preview: Preview = {
+  ...basePreview,
+  parameters: {
```

Comment:
does it not matter if the "parameters" are inside teh "createPreview"?
else you could change basePreview to be a function where you can pass in the additional parameters.

Reply:
ah ok, so that was intentional. got it!